build.gradle.kts

```diff
@@ -1,162 +1,4 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 plugins {
-    alias(libs.plugins.kotlin.jvm)
-    jacoco
-    alias(libs.plugins.detekt)
+    alias(libs.plugins.kotlin.jvm) apply false
     alias(libs.plugins.dokka)
```

Comment:
Just out of curiosity, why do these need to stay in the original `build.gradle` file?

Reply:
Great question:
- the `alias(libs.plugins.kotlin.jvm)` allows other buildscripts to use `kotlin`-specific Gradle commands (like applying `plugins { kotlin("kapt") }`) without having to specify a version, because that is configured here. It's `apply false` though, since we don't actually need any Kotlin-specific things in the main `build.gradle` file.

- the `alias(libs.plugins.dokka)` allows using the project-wide `dokkaHtmlMultiModule` that's used for docs. In the "purest" Gradle sense, this should probably be more like the `jacoco` sub-module and removed from the root `build.gradle` file to work towards the goal of not having cross-project configuration happening: https://gradle.github.io/configuration-cache/

Reply:
OK cool, makes sense 👍